### ISSUE TITLE
vignette issues

### ISSUE THREAD

**Issue by josherrickson** [benbhansen-stats/propertee#70]
While rebuilding the reference site, I ran into issues with two vignettes.

1) covadj_with_clustering: I made several tweaks, mostly around `type` argument being HC0 vs CR1. There was also two tests I couldn't make pass so I commented them out. See d681f704e for the details.
2) polynomialSimulation just wouldn't run so I changed the extension to avoid building it.

**Comment by benthestatistician** [comment]
Can we figure out a way to include the polynomialSimulation vignette on the reference site?  Perhaps with caching of results, to avoid time-consuming simulations whenever the site is rebuilt.

**Comment by josherrickson** [comment]
I'd made some pretty sweeping changes to the vignette structure. I've summarized the new structure on the wiki (https://github.com/benbhansen-stats/propertee/wiki/Vignettes,-articles,-and-otherwise), but the general idea is to separate documents into three categories:

- vignettes for CRAN & website
- articles for website only
- developer documents

Both vignettes and articles should go in /vignettes; for articles, add manual entries to .Rbuildignore (no longer using /not-for-cran folder).

Developer documents go in /developer_docs.

I know there are a few things inside /developer_docs that need to go back into vignettes (mi_vignette and polynomial_sim specifically), but they're currently in deverloper_docs because they need additional work and I wanted to get the repo cleaned up first.

**Comment by josherrickson** [comment]
If there's demand for automation in building /developer_docs, I can setup a Makefile.

**Comment by josherrickson** [comment]
I've re-added the mi_vignette. @jwasserman2 feel free to tweak here: https://github.com/benbhansen-stats/propertee/blob/7ccf9d6e428b2347adf64081a70ca578b034eedf/_pkgdown.yml#L22-L23

either the name of the link, or it's placement in the menu. (Should we create a new Examples section?)

Also, I did not exclude this from CRAN (it passes CRAN checks right now, though I haven't done the more rigorous ones yet), I think I'd prefer to exclude it just to avoid any issues with the additional packages, any thoughts @benthestatistician @jwasserman2 ?

**Comment by josherrickson** [comment]
I take it back - it builds locally, but fails on the server. I believe it's having trouble getting the remote data. Can we modify it to keep the data locally? (We can have non-executed code showing loading downloading it, but have a hidden code chunk running locally?)

**Comment by benthestatistician** [comment]
1. I'm OK w/ having this one be website only, to limit CRAN complications.
2. I'm also OK with our keeping the data locally, but only if we can do that without having to bundle it with the package tarball.  I would also prefer to avoid saving it (particularly the `2011-2014MME.zip` file from www.michigan.gov/cepi/) to a public github site.

**Comment by josherrickson** [comment]
Good catch - we were git ignoring vignettes/*.R and when I moved the script out of vignettes/ and back in again, it got lost. Reincluded it

**Comment by josherrickson** [comment]
And well wouldn't you know it, that was the only issue. The MI script is back online. @jwasserman2 same offer to tweak menu placement or name if you want. 

I'll turn to the poly sim when I get a chance.

**Comment by jwasserman2** [comment]
Nice! It looks good to me the way it is. Thank you to @Cassandra291201 for pointing out the bug
